src/java.desktop/share/native/libawt/awt/image/awt_parseImage.c

```diff
@@ -454,6 +454,7 @@ int awt_parseColorModel (JNIEnv *env, jobject jcmodel, int imageType,
 
     int i;
     static jobject s_jdefCM = NULL;
+    cmP->nBits = NULL;
```

Comment:
I think you could leave the cmP->nBits initialization where it was. But that's just a nit.